src/time-picker/demos/enUS/step-time.demo.md

```diff
@@ -0,0 +1,22 @@
+# Step Time
+
+Pass a number as step or use an array to specify the item you want.
+
+```html
+<n-time-picker
+  v-model:value="time0"
```

Comment:
remove the v-model (and the script part), use uncontrolled mode.
the demo will be more clear